msgraph-developer-proxy/ProxyCommandHandler.cs

```diff
@@ -31,8 +34,11 @@ public class ProxyCommandHandler : ICommandHandler {
 
     public async Task<int> InvokeAsync(InvocationContext context) {
         int port = context.ParseResult.GetValueForOption(Port);
-        CancellationToken? cancellationToken = (CancellationToken?)context.BindingContext.GetService(typeof(CancellationToken?));
         Configuration.Port = port;
+        LogLevel logLevel = context.ParseResult.GetValueForOption(LogLevel);
+        _logger.LogLevel = logLevel;
```

Comment:
As alluded to in the ProxyHost, we should be null checking logLevel and only assigning here if the user supplies a value.

As I think of it now, port probably needs the same treatment.

Reply:
Good catch! I'll adjust both.